util/test/chpl_launchcmd.py

```diff
@@ -127,6 +127,7 @@ def full_test_command(self):
             full_test_command += ['test', '-f', self.test_command[0], '&&']
 
         full_test_command.extend(self.test_command)
+        full_test_command.extend(['&>{0}'.format(output_file)])
```

Comment:
I imagine this works since you've tested it, but isn't `&> {0}` the `csh` notation for stdout/stderr redirection?

For `bash`, it's generally `> {0} 2>&1`